scripts/check-pkg-for-release.js

```diff
@@ -162,10 +172,13 @@ function checkDependencyAddedInMDCPackage() {
 }
 
 function checkPkgDependencyAddedInMDCPackage() {
-  assert.notEqual(typeof MASTER_PACKAGE_JSON.dependencies[CLI_PACKAGE_JSON.name], 'undefined',
-    'FAILURE: Component ' + CLI_PACKAGE_JSON.name + ' is not a dependency for MDC Web. ' +
-    'Please add ' + CLI_PACKAGE_JSON.name +' to ' + MASTER_PACKAGE_JSON_RELATIVE_PATH +
-    '\' dependencies before commit.');
+  const name = getPkgName();
+  if (!CSS_WHITELIST.filter((x) => JS_WHITELIST.includes(x)).includes(name)) {
```

Comment:
Same comment as above: favor early exits

Reply:
Just saw this - will make the changes in the new PR